README.md

```diff
@@ -1,164 +1,103 @@
-# lamp
+# LAMP
 Lightweight Application for Measuring Performance
 
-### Setup
+## Architecture
+![LAMP Architecture Diagram](architecture.jpg)
+[Link](https://miro.com/app/board/uXjVOzXKW9s=/?share_link_id=356679616715) to
+Miro Diagram
+
+### GTFS to Parquet
+Raw Real Time GTFS files are collected in an incoming S3 bucket and are
+populated by [Delta](https://github.com/mbta/delt), a small service that logs
+http files to the bucket configurably. A python based lambda collects all of the
+files in the incoming bucket and organizes them into batch of similar types.
+These batches are then processed by a second python based lambda that adds all
+of the data in each real time gtfs file into a single parquet file. These
+parquet files are then uploaded to a separate S3 bucket, with all processed
+files moved to an archive.
+
+The python module used for batching and conversions is located in
+`/py_gtfs_rt_ingestion/`, along with its tests and the scripts that are
+triggered in each of the lambda functions. This modules, its dependencies, and
+the scripts are zipped up together before being deployed to the lambda
+functions.
+
+#### Types of GTFS Files
+* Real Time Alerts
+* Real Time Bus Trip Updates
+* Real Time Bus Vehicle Positions
+* Real Time Trip Updates
+* Real Time Vehicle Count
+* Real Time Vehicle Positions
+* Schedule Data
+
+Import schemas for these file types can be found [here](import_schemas.md).
 
-Run the following:
+### Parquet to Relational Database
+The joined parquet files are going loaded into an RDS using Amazon Glue.
```

Comment:
AWS Glue, or a similar service, will be used to concatenate and join the parquet files. This will create intermediate data representations that will be loaded into an RDS for low latency access.